packages/api/package.json

```diff
@@ -62,12 +62,12 @@
     "@typescript-eslint/parser": "^5.0.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^4.0.0",
-    "jest": "28.1.3",
+    "jest": "29.2.1",
```

Comment:
Update `jest` to fix an issue with open handlers for supertest request post functions.